statsmodels/base/tests/test_generic_methods.py

```diff
@@ -721,13 +710,12 @@ def initialize(cls):
         cls.res = mod.fit(cov_type='HC0')
 
 
-class T_estWaldAnovaOLSNoFormula(object):
+@pytest.mark.skip(reason="TODO: Find a reason.  "
+                         "This used to just have a mangled name.")
+class TestWaldAnovaOLSNoFormula(object):
```

Comment:
@josef-pkt this was introduced in #2132 (yours) and was never not-mangled.  That PR doesn't have any comments that suggest why this was done.  Any recollection?  (Also, this and several other test-improvement PRs would be super-swell to get out of the way (plus a couple of 1-line-diff PRs by bashtage and a new contributor, respectively))

Reply:
looks like I changed my mind during a commit and added a test method instead of a test class.
This class is incomplete and if I make it follow the pattern of other classes, then both inherited methods fail, 
i.e. CheckAnovaMixin only works for models with formula
https://github.com/statsmodels/statsmodels/pull/2132/commits/5270986bbef7d22a7b595e69797ef5dbd6b4eaf1

this test class can be deleted

Reply:
updated.